src/browser/fixture.js

```diff
@@ -29,22 +29,29 @@ function getComposedChildren(node) {
 
 }
 
-async function waitForElem(elem) {
+async function waitForElem(elem, opts) {
 
 	const update = elem?.updateComplete;
 	if (typeof update === 'object' && Promise.resolve(update) === update) {
 		await update;
 		await nextFrame();
 	}
 
+	if (opts.awaitLoadingComplete && typeof elem?.getLoadingComplete === 'function') {
+		await elem.getLoadingComplete();
+		await nextFrame();
+	}
+
 	const children = getComposedChildren(elem);
-	await Promise.all(children.map(e => waitForElem(e)));
+	await Promise.all(children.map(e => waitForElem(e, opts)));
 
 }
 
 export async function fixture(element, opts) {
+	opts ??= {};
+	opts.awaitLoadingComplete ??= true;
```

Comment:
Just pointing out that this is mutating `opts`, so back in the test suddenly `opts` has this value. Probably not really a big deal here, but my default would be to do:
```javascript
waitForElem(elem, { awaitLoadingComplete: true, ...opts });
```

This also means it doesn't matter was `opts` is, whereas currently `fixure(el, 1)` would fail to set `awaitLoadingComplete`.

Reply:
I like your suggestion much better, thanks!

Reply:
Adding to that... does `waitForElem` need the other opts passed in, or do we think it might eventually? Could this just be
```
await waitForElem(elem, { awaitLoadingComplete: opts.awaitLoadingComplete ?? true });
```
Or simply a boolean parameter if we don't think other options will be needed?

Reply:
I like the boolean parameter actually -- switched it to that. It's private, so if we need more options later we can always add them.